Makefile

```diff
@@ -80,6 +80,8 @@ generate-elm-test-json:
 	@cd go && go test -run TestGenerateGoFixtures > /dev/null 2>&1
 	@echo "Generating cross-validation test data..."
 	node scripts/generate-cross-validation-tests.js
+	@echo "Formatting generated Elm test data..."
+	npx elm-format tests/GoElmCrossValidationTestData.elm --yes
```

Comment:
Added automatic formatting after test generation to keep git history clean.

The Node.js generator doesn't produce properly formatted Elm code. Without this, every `make test-json-compatibility` would create formatting-only diffs.